### ISSUE TITLE
some questions about training

### ISSUE THREAD

**Issue by lightein** [HiKapok/SSD.TensorFlow#19]
Thanks for your works, it is reproduced easily. There are still some questions confusing me.

1. The pre-trained vgg16_reduced_fc is different the orginal vgg16, including different weights on conv1/conv1_1 and reducing the fc8 layer, why?

2. The learning rate is a piecewise function. how did you design this function? and how to set the initial learning rate?

 3. i trained the net in a pc with two 1080ti. the tow gpu is both used. After 120000 steps, the mAP is 0.78. And the log print in the screen like that:

INFO:tensorflow: global_step/sec: 2.22489

INFO: tensorflow: lr=0.00001, ce= 1.487573, loc=0.481032, loss=7.578554, l2=5.609949, acc= 0.835833

INFO: tensorflow: loss=10.1866, step=119990(4.494 sec).

there are two data named "loss", what is their difference?

suppose the second "loss" is the total loss just as the file loss.JPG saved in the log directory. why total loss in your loss.JPG is converged about 7\~8. Then my is converged 9\~10.

And my time of training one batch cost more than you(yours are 1s) why?

please help me.

**Comment by HiKapok** [comment]
@lightein 1. You can refer to the paper about FCN for more details. 2. All the lr settings follows the official codes. 3. the final loss of the last line should denotes all the losses that been added into loss collection, I re-weight some loss of those. Maybe the communication between GPUs is the problem

**Comment by lightein** [comment]
@HiKapok Thanks a lot. Did you modify the vgg16?  Vgg16/conv1/conv1_1/weights in your reduced version is different from the origin vgg16. Others are same.
